credoai/evaluators/model_profiler.py

```diff
@@ -0,0 +1,202 @@
+from typing import Optional
+
+from pandas import DataFrame
+
+from credoai.evaluators import Evaluator
+from credoai.evaluators.utils.validation import check_existence
+from credoai.evidence.containers import ModelProfilerContainer
+from credoai.utils import ValidationError, global_logger
+
+USER_INFO_TEMPLATE = {
+    "developed_by": None,
+    "shared_by": None,
+    "model_type": None,
+    "intended_use": None,
+    "downstream_use": None,
+    "out_of_scope_use": None,
+    "language": None,
+    "related_models": None,
+    "license": None,
+    "resources_for_more_info": None,
+    "input_description": None,
+    "output_description": None,
+    "performance_evaluated_on": None,
```

Comment:
Yeah my idea here was more like providing examples rather than enforcing types. I imagined this to be strings for the most part, they could also contain Markup for example.
I wanted the user to be free to experiment/add to it